src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -318,14 +324,14 @@ public int RenameTrackingPreview
 
         public int ShowKeywords
         {
-            get { return GetBooleanOption(CompletionOptions.IncludeKeywords); }
-            set { SetBooleanOption(CompletionOptions.IncludeKeywords, value); }
+            get { return 0; }
+            set { }
         }
 
         public int ShowSnippets
         {
-            get { return GetBooleanOption(CSharpCompletionOptions.IncludeSnippets); }
-            set { SetBooleanOption(CSharpCompletionOptions.IncludeSnippets, value); }
+            get { return this.SnippetsBehavior; }
```

Comment:
This is subtly breaking anyone who was using this before and expecting 0 to mean "No snippets" and 1 to mean "Snippets".